css-brochure-form/styles.css

```diff
@@ -0,0 +1,58 @@
+h2 {
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 100;
+  font-size: medium;
+  color: #c85a53;
+}
+
+form {
+  text-align: center;
+  margin-top: 150px;
+}
+
+.email {
+  margin-right: 305px;
+}
+
+.hiring {
+  margin-right: 270px;
+}
+
+.name {
+  margin-right: 360px;
+}
+
+.amount {
+  margin-right: 70px;
+}
```

Comment:
You want to avoid giving such high `margin` values. By having such high values, it means you're doing something "hacky" to get this to work. If you are trying to center everything, tell your `body` to have a `max-width: 500px` or something, and use `margin: 0 auto`